Source/WebInspectorUI/UserInterface/Views/DOMBreakpointTreeElement.css

```diff
@@ -23,6 +23,10 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+body:not(.window-inactive, .window-docked-inactive) .tree-outline:focus-within .item.breakpoint.dom.selected .status > .status-image {
```

Comment:
Any reason to not just do a single rule in `BreakpointTreeElement.css` as something like `body:not(.window-inactive, .window-docked-inactive) .tree-outline:focus-within .item.breakpoint.selected .status > .status-image`?

Reply:
JS breakpoints require that the breakpoint be "resolved" (i.e. translate the line/column used by the frontend to an actual pause position in JSC) in order to have the white outline
```
body:not(.window-inactive, .window-docked-inactive) .tree-outline:focus-within .item.breakpoint.javascript.selected .status > .status-image.resolved {
    stroke: var(--selected-foreground-color);
}
```

Reply:
tho actually looking at this more, it seems like all non-JS breakpoints should be "resolved" by default (according to `WI.Breakpoint.prototype.get resolved`), so we might just need to move that rule to be more general
```
body:not(.window-inactive, .window-docked-inactive) .tree-outline:focus-within .item.breakpoint.selected .status > .status-image.resolved {
    stroke: var(--selected-foreground-color);
}
```
i'll give that a shot